ubuntu.18.04/Dockerfile

```diff
@@ -1,24 +1,23 @@
 FROM ubuntu:18.04
 
-ARG Powershell_Version=7.0.0\*
-ARG Octopus_Cli_Version=7.3.2
-ARG Octopus_Client_Version=8.4.0
-ARG Azure_Cli_Version=2.4.0\*
-ARG Azure_Powershell_Version=2.2.0
-ARG Helm_Version=v3.2.4
+ARG Powershell_Version=7.0.3\*
+ARG Octopus_Cli_Version=7.4.1
+ARG Octopus_Client_Version=8.8.3
+ARG Azure_Cli_Version=2.10.1\*
+ARG Azure_Powershell_Version=4.5.0
+ARG Helm_Version=v3.3.0
 ARG Node_Version=12.18.3\*
-ARG Kubectl_Version=1.18.6-00
-ARG Terraform_Version=0.12.24
-ARG Eks_Cli_Version=0.23.0
-ARG Ecs_Cli_Version=1.18.1
-ARG Aws_Iam_Authenticator_Version=1.16.8

```

Comment:
This is not actually the version of the aws-iam-authenticator - its the version of k8s instead.